### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nictownsend**
Closes #0

Fix scss snippets that do not compile

#### Changelog

**Added**

**Changed**

- README scss snippets that do not compile

**Removed**

#### Testing / Reviewing

Manually try out the changes

### CODE REVIEW

**Comment by nictownsend** [comment]
I have read the DCO document and I hereby sign the DCO.

File: packages/themes/README.md [hunk 1]
```diff
@@ -58,14 +58,16 @@ Inline theming can be done by using the mixin. For example:
 // Use the default white theme here
 
 .my-dark-theme {
-  @include theme.theme(themes.$g90) {
-    // Use the dark theme here
+  @include theme.theme(themes.$g90)
+  &__something {
+    //any theme references in here, or in any nested classes/components will resolve to g90
   }
 }
```
Comment (tw15egan) [inline packages/themes/README.md]: Can we just do 

```scss
.my-dark-theme {
  @include theme.theme(themes.$g90);
}
```

https://stackblitz.com/edit/github-e91zc8?file=src%2Findex.scss,src%2FApp.jsx,package.json

**Review by tay1orjones — APPROVED** [review]
Thanks for improving this!

**Review by tw15egan — APPROVED** [review]
Thanks for the improvement 💪🏻

**Pull request merged**